src/components/footer/Footer.js

```diff
@@ -5,20 +5,22 @@ import Button from 'components/button/Button';
 
 // @TODO: Bring it back when ready
 // import DonateButton from './DonateButton';
-import styles from './footer.module.scss';
 
-function Footer({ onClickLegal, t }) {
```

Comment:
Removed on purpose?

Reply:
thanks @tureey I removed the `t` that was being passed in to the Footer function as it wasn't being called from anywhere.